docs/data/date-pickers/date-time-picker/date-time-picker.md

```diff
@@ -0,0 +1,61 @@
+---
+title: React Date Time Picker component
+components: DateTimePicker,DesktopDateTimePicker,MobileDateTimePicker,StaticDateTimePicker
+githubLabel: 'component: DateTimePicker'
+packageName: '@mui/x-date-pickers'
+materialDesign: https://material.io/components/date-pickers
+---
+
+# Date Time Picker
+
+<p class="description">Combined date & time picker.</p>
+
+This component combines the date & time pickers. It allows the user to select both date and time with the same control.
+
+Note that this component is the [DatePicker](/components/pickers/date-picker/) and [TimePicker](/components/pickers/time-picker/)
```

Comment:
I don't know why, but in the peview: this link redirect to "/material-ui/pickers/date-picker/"

Reply:
That's the logic from the core repo. It should be fine when we enable the redirects.

Reply:
Good catch, I did not updated the links in the pages
The pickers docs will never be released on the old structure, so we can change them

Reply:
https://github.com/mui/mui-x/pull/3451/commits/7d74bfb0afca8766f3baef95c86264682c37a53b

I think I got them all